src/Symfony/Component/Routing/Loader/schema/routing/routing-1.0.xsd

```diff
@@ -87,16 +87,16 @@
 
   <xsd:complexType name="map">
       <xsd:choice minOccurs="0" maxOccurs="unbounded">
-          <xsd:element name="boolean" nillable="true" type="map-boolean-entry" />
-          <xsd:element name="integer" nillable="true" type="map-integer-entry" />
+          <xsd:element name="bool" nillable="true" type="map-bool-entry" />
+          <xsd:element name="int" nillable="true" type="map-integer-entry" />
```

Comment:
If we agree on renaming the types, we can do that here too (even if these are internal details that are not exposed to the end user).